packages/@internationalized/date/docs/ZonedDateTime.mdx

```diff
@@ -43,7 +43,7 @@ let date = new ZonedDateTime(
   // Time zone and UTC offset
   'America/Los_Angeles', -28800000,
   // Time
-  12, 24, 45
+  9, 15, 00
```

Comment:
Nitpick: linters will catch this as a potential octal literal which is problematic in es5+

```suggestion
  9, 15, 0
```